web/elm/src/FlySuccess/FlySuccess.elm

```diff
@@ -14,7 +14,7 @@ import FlySuccess.Models as Models exposing (ButtonState(..), Model, hover)
 import FlySuccess.Styles as Styles
 import FlySuccess.Text as Text
 import Html exposing (Html)
-import Html.Attributes exposing (attribute, href, id, style)
+import Html.Attributes exposing (attribute, class, href, id, style, value)
```

Comment:
Heyyy, if you run the `yarn analyse -s` and go to `localhost:3000`, you will see there are two error messages; here we need to delete the unused `class`.